CodeChallengeTests/Base/CodeChallengeTestCase.swift

```diff
@@ -33,27 +33,24 @@ class CodeChallengeTestCase: XCTestCase {
     }
     
     fileprivate func printResults<ChallengeType: CodeChallengeType>(_ results: [AccumulatedChallengeResult<ChallengeType>]) {
-        let maxNameLength = results.reduce(0) { max($0, $1.name.characters.count) }
         let formatter = NumberFormatter()
         formatter.numberStyle = .decimal
         formatter.minimumFractionDigits = 5
         formatter.maximumFractionDigits = 5
 
-        for (i, accResult) in results.enumerated() {
-            var name = accResult.name
-            // try to make the results line up a bit by inserting the correct amount of tabs between the name and the avg based on the maximum name length vs this name's length
-            if name.characters.count < maxNameLength {
-                let diff = maxNameLength - name.characters.count
-                let tabWidth = 5
-                var tabsToInsert = Int(ceil(Double(diff)/Double(tabWidth)))
-                if diff % tabWidth == 0 {
-                    tabsToInsert += 1
-                }
-                for _ in 1...tabsToInsert {
-                    name.append(Character("\t"))
-                }
+        var names = [String]()
+        for (i, result) in results.enumerated() {
+            names.append("\(i+1). \(result.name)")
```

Comment:
assuming you're referring to the space after the dot? That's hard to see on gh, but it's inside a string, so yes. It's the period after the number, so the space is required before the name.